example/lib/main.dart

```diff
@@ -85,10 +85,25 @@ class _MyHomePageState extends State<MyHomePage> {
         leading: DescribedFeatureOverlay(
           featureId: feature1,
           tapTarget: icon2,
-          backgroundColor: Colors.teal.shade800,
+          backgroundColor: Colors.teal,
           title: 'Just how you want it',
           description:
-              'Tap the menu icon to switch accounts, change settings & more.',
+              Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: <Widget>[
+                  const Text('Tap the menu icon to switch accounts, change settings & more.'),
+                  const SizedBox(height: 12),
+                  FlatButton(
+                    padding: const EdgeInsets.all(0),
+                    child: Text(
+                      "UNDERSTOOD", 
```

Comment:
I think that this does not need to be all uper case because `FlatButton` does that automatically.
```suggestion
                      'Understood', 
```